test/code/plugins/in_npmd_server_plugintest.rb

```diff
@@ -3,6 +3,9 @@
 require 'socket'
 require 'securerandom'
 
+# TODO: Correct test cases 6,7,8 and check errors in execution of test cases 15, 16
```

Comment:
Actually need to remove that. Local code checkin had that TODO which is completed.